<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSetupExportVTK</class>
 <widget class="QDialog" name="DSetupExportVTK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>176</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;This dialog shows the Burgers vector of the dislocation as guessed by ddplot or previously entered manually. The VTK exported from ddplot will contain relative displacements between atoms projected in the direction parallel to the Burgers vector (Zpar) and into the direction perpendicular to the Burgers vector (Xperp, Yperp). Because the latter is not uniquely determined from the knowledge of the direction of the Burgers vector, it has to be inserted below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="textLabel3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Burgers vector of the dislocation (relative to &lt;100&gt;):</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="b2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="b3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="b1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel component of projected relative atomic displacements - this is typically the Burgers vector (relative to &lt;100&gt;):</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="zpar1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zpar2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zpar3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>269</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Perpendicular component of projected relative atomic displacements - two vectors perpendicular to Zpar (relative to &lt;100&gt;):</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Zpar =</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Orientation of the atomic block (relative to &lt;100&gt;):</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="x1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X =</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="x2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y =</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="z1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z =</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="textLabel3_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yperp =</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="xperp2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel3_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Xperp =</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="xperp3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xperp1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yperp2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yperp3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yperp1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <zorder>textLabel4</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>textLabel3_5</zorder>
  <zorder>xperp2</zorder>
  <zorder>textLabel3_4</zorder>
  <zorder>xperp3</zorder>
  <zorder>xperp1</zorder>
  <zorder>yperp2</zorder>
  <zorder>yperp3</zorder>
  <zorder>yperp1</zorder>
 </widget>
 <tabstops>
  <tabstop>x1</tabstop>
  <tabstop>x2</tabstop>
  <tabstop>x3</tabstop>
  <tabstop>y1</tabstop>
  <tabstop>y2</tabstop>
  <tabstop>y3</tabstop>
  <tabstop>z1</tabstop>
  <tabstop>z2</tabstop>
  <tabstop>z3</tabstop>
  <tabstop>b1</tabstop>
  <tabstop>b2</tabstop>
  <tabstop>b3</tabstop>
  <tabstop>zpar1</tabstop>
  <tabstop>zpar2</tabstop>
  <tabstop>zpar3</tabstop>
  <tabstop>xperp1</tabstop>
  <tabstop>xperp2</tabstop>
  <tabstop>xperp3</tabstop>
  <tabstop>yperp1</tabstop>
  <tabstop>yperp2</tabstop>
  <tabstop>yperp3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DSetupExportVTK</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DSetupExportVTK</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
